packages/xo-server/src/api/backup-ng.js

```diff
@@ -68,7 +68,7 @@ export function editJob(props) {
 editJob.permission = 'admin'
 editJob.params = {
   compression: {
-    enum: ['', 'native'],
+    enum: ['', 'native', 'zstd'],
```

Comment:
I remember why I added `''`, it's because `undefined` means not editing the `compression` setting of the job.